tests/backup/backup_test.go

```diff
@@ -8097,6 +8097,24 @@ func GetAllBackupsAdmin() ([]string, error) {
 	return backupNames, err
 }
 
+// GetAllRestoresAdmin returns all the backups that px-central-admin has access to
+func GetAllRestoresAdmin() ([]string, error) {
+	restoreNames := make([]string, 0)
+	backupDriver := Inst().Backup
+	ctx, err := backup.GetAdminCtxFromSecret()
+	log.FailOnError(err, "Fetching ctx")
```

Comment:
Lets not `log.FailOnError()` here, this function returns error, so lets just return it here

Reply:
Done. Fixed it.